core/src/main/scala/astraea/spark/rasterframes/stats/CellHistogram.scala

```diff
@@ -49,6 +51,98 @@ case class CellHistogram(stats: CellStatistics, bins: Seq[CellHistogram.Bin]) {
 
     lines.mkString("\n")
   }
+
+  // find the count of the bin that the value fits into
```

Comment:
Change this to ScalaDoc comment, using `/**`  and `*/ ` delimiters. Also add comment about what happens when `label` is not exactly represented in the bins.